core/src/main/res/values/strings.xml

```diff
@@ -33,6 +34,10 @@
     <string name="recently_published_episodes_label">Recently published</string>
     <string name="episode_filter_label">Show only new Episodes</string>
 
+    <!-- Statistics fragment -->
+    <string name="total_time_listened_to_podcasts">Total time listened to podcasts:</string>
```

Comment:
Nitpick:
In fact, this might not be true. I listen to podcasts at a speed of 1.75x. I only listened about ``totalTime/1.75`` hours...
(But I can't come up with a better text at the top of my head, either)